env-example

```diff
@@ -70,3 +70,7 @@ APPLE_PROVIDER_ID=
 APPLE_KEY_ID=
 APPLE_KEY_PEM_B64=
 APPLE_API_BRIDGE_URL=
+
+# slack notifications
+SLACK_SNS_TOPIC=
```

Comment:
when it comes time to add these variables in to stag/prod, i forget how that is done?

Reply:
You'll need to wire through the SNS topic to the Infrastructure feeder.yml stack, and set the channel id.  Here's that same change for Augury - https://github.com/PRX/Infrastructure/commit/8bf3c3d787ee549cbc252906c573c2f554f4e630